packages/sanity/src/core/i18n/simpleParser.ts

```diff
@@ -28,6 +29,24 @@ function matchCloseTag(input: string) {
   return input.match(CLOSE_TAG_RE)
 }
 
+function validateTagName(tagName: string) {
+  const isValidComponentName = VALID_COMPONENT_NAME_RE.test(tagName)
+  if (isValidComponentName) {
+    return
+  }
+
+  const isValidHtmlTagName = VALID_HTML_TAG_NAME_RE.test(tagName)
+  if (isValidHtmlTagName) {
+    return
+  }
+
+  throw new Error(
+    tagName.trim() === tagName
+      ? `Invalid tag "<${tagName}>". Tag names must be lowercase HTML tags or start with an uppercase letter and can only include letters and numbers.`
```

Comment:
re:previous comment: maybe this would be a better place to mention what subset of tags we allow?

Reply:
Maybe. It does print an error in the Translate component, so I think that is fine for now. I tried refactoring, but ran into some parser shenanigan holes that I don't want to reason about at the moment. We can get back to it later if it becomes a problem.